app/helpers/application_helper.rb

```diff
@@ -0,0 +1,15 @@
+module ApplicationHelper
+    def user_can_edit(user_id)
```

Comment:
Instead of redeclaring this here, you could make it a helper_method in ApplicationController  (leaving aside for now whether it's the right name...)